vault/core.go

```diff
@@ -1761,15 +1768,25 @@ func (c *Core) runStandby(doneCh, stopCh, manualStepDownCh chan struct{}) {
 		close(keyRotateStop)
 		<-keyRotateDone
 		close(checkLeaderStop)
+		c.logger.Trace("core: closed periodic leader refresh channel")
 		<-checkLeaderDone
+		c.logger.Trace("core: periodic leader refresh returned")
 	}()
 
+	var manualStepDown bool
 	for {
 		// Check for a shutdown
 		select {
 		case <-stopCh:
+			c.logger.Trace("core: stop channel triggered in runStandby")
```

Comment:
Since we are no longer closing the stop channel we should also add a case here that checks if lockAcquisitionStopCh is closed.